salt/elife-xpub/init.sls

```diff
@@ -53,6 +53,8 @@ elife-xpub-environment-variables-for-configuration:
             export MAILER_PORT={{ pillar.elife_xpub.mailer.port }}
             {% if pillar.elife_xpub.mailer.user %}
             export MAILER_AUTH='{"user": "{{ pillar.elife_xpub.mailer.user }}", "pass":"{{ pillar.elife_xpub.mailer.pass }}"}'
+            export MAILER_AUTH_USER={{ pillar.elife_xpub.mailer.user }}
+            export MAILER_AUTH_PASS={{ pillar.elife_xpub.mailer.pass }}
```

Comment:
this is a base64 value so can contain `+`, `/` and `=`. These all seem safe without quoting.

Reply:
Ah - do you think I should quote just to be on the safe side?